### ISSUE TITLE
Download of plugins fails when behind proxy

### ISSUE THREAD

**Issue by Schmytzi** [askimed/nf-test#265]
I am working on a system behind a proxy. Every time I try to run `nf-test test`, it fails with the following output:
```
🚀 nf-test 0.9.0
https://www.nf-test.com
(c) 2021 - 2024 Lukas Forer and Sebastian Schoenherr

Error: Syntax errors in nf-test config file: java.net.ConnectException: Connection refused
```
This error also occurs with `nf-test update-plugins`.

This does not appear to be an issue with the nf-test configuration, as the tests run fine elsewhere. It is also possible to circumvent this error by first running the tests on a computer with direct internet access and then copying the the directory `.nf-test/plugins` to the affected system but this seems more like a dirty hack.

**Comment by MarieLataretu** [comment]
Hi there, I have the same problem, see #234 !

My workaround was to download the jar files from the plugin repos (`wget` worked in my case) and add them in the test config (see [here](https://www.nf-test.com/docs/plugins/using-plugins/)), or use `--plugin path/to/my/nft-plugin.jar:path/to/my/nft-plugin2.jar`

**Comment by Schmytzi** [comment]
Has been fixed with PR #264. Now it's enough to run the following before starting nf-test
```bash
export NFT_JAVA_ARGS='-Djava.net.useSystemProxies=true'
```
